### ISSUE TITLE
"Get Started" link on the main page doesn't walk you through code examples

### ISSUE THREAD

**Issue by slightlyoff** [Polymer/docs#289]
The first 2 links from the home page ("Get Polymer" and "Get Started") seem like they'd be related, but "Get Started" links to [this page](http://www.polymer-project.org/docs/start/everything.html) which doesn't start with a code example, doesn't talk about anything related to the code you might have just gotten, and doesn't have examples that you can cut/paste that show you how Polymer makes a page awesome.

In short, lots of telling, very little showing.
